packaging/opensuse/snapd.spec

```diff
@@ -35,12 +46,6 @@
 %{?!_systemd_system_env_generator_dir: %global _systemd_system_env_generator_dir %{_prefix}/lib/systemd/system-environment-generators}
 %{?!_systemd_user_env_generator_dir: %global _systemd_user_env_generator_dir %{_prefix}/lib/systemd/user-environment-generators}
 
-# This is fixed in SUSE Linux 15
-# Cf. https://build.opensuse.org/package/rdiff/Base:System/rpm?linkrev=base&rev=396
-%if 0%{?suse_version} < 1500
-%global _sharedstatedir %{_localstatedir}/lib
-%endif
```

Comment:
Removing this will break things, unless you intend to drop everything older than Leap 15.0.

Reply:
It works because I stopped using sharedstatedir. I did this because it was defined differently in RPM (see what the makefile does) and in the spec file (because of the override).

In the end, this doesn't bring us any value, killing it makes the code simpler.

Reply:
It _does_ buy me something, the ability to compare the file lists between the Fedora and openSUSE packages.

And it isn't defined differently anymore, since I fixed that in openSUSE, only in older ones.

Reply:
It is defined differently because if you ask RPM alone it won't now anything that is defined in the spec. I did this for a reason. This allows us to save considerable copy pasted code across packaging.

You can also use localstatedir in Fedora, sharedstatedir doesn't buy us anything. It's not like we either change it or support values other than /var/lib.

Reply:
We can look at undoing this now. I've implemented the idea with `snapd.defines.mk`

Reply:
I am happy to undo this with a follow up.

Reply:
Can we please put this back now, since you've got the defines makefile now?